src/modules/course/schema/enrolledCourse.schema.ts

```diff
@@ -0,0 +1,41 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import * as mongoose from 'mongoose';
+
+export type EnrolledCourseDocument = EnrolledCourse & Document;
+
+interface video {
+  num: number;
+  timestamp?: Date;
+}
+
+@Schema()
+export class EnrolledCourse {
+  @Prop({})
+  eId: mongoose.Schema.Types.ObjectId;
```

Comment:
Why you have created this eID, 
Ids are auto-generated by mongo.

Reply:
I had put it for testing on postman with the correct response
i will remove it now